database/seed.js

```diff
@@ -1,9 +1,5 @@
 const { Sequelize, DataTypes } = require('sequelize');
 const schema = require('./index.js');
-const sequelize = new Sequelize('availability', 'root', 'sqlcraft', {
-  host: 'localhost',
-  dialect: 'mysql'
-})
 
 //Verify connection
 const auth = async ()=> {
```

Comment:
I noticed this same function was in your index.js file. It doesn't look like it's being invoked in this file so try deleting it and then see if your seeding script still works.